sympy/stats/crv.py

```diff
@@ -444,12 +444,17 @@ def probability(self, condition, **kwargs):
         except NotImplementedError:
             from sympy.stats.rv import density
             expr = condition.lhs - condition.rhs
-            dens = density(expr, **kwargs)
+            if len(random_symbols(expr)) == 0:
```

Comment:
IMO, `if not random_symbols` can also work.

Reply:
Will change that